src/Sentry.Unity.Editor/Native/BuildPostProcess.cs

```diff
@@ -40,43 +41,83 @@ public static void OnPostProcessBuild(BuildTarget target, string pathToProject)
                     return;
                 }
 
-                var crashpadPath = Path.GetFullPath(Path.Combine("Packages", SentryPackageInfo.GetName(), "Plugins",
-                    "Windows", "Sentry", "crashpad_handler.exe"));
-                var targetPath = Path.Combine(Path.GetDirectoryName(pathToProject), Path.GetFileName(crashpadPath));
-                File.Copy(crashpadPath, targetPath, true);
+                var projectDir = Path.GetDirectoryName(executablePath);
+                addCrashHandler(logger, projectDir);
+                uploadDebugSymbols(logger, projectDir, Path.GetFileName(executablePath));
 
-                // TODO symbol upload
-                // var sentryCliOptions = SentryCliOptions.LoadCliOptions();
-                // if (!sentryCliOptions.UploadSymbols)
-                // {
-                //     logger.LogDebug("Automated symbols upload has been disabled.");
-                //     return;
-                // }
-                //
-                // if (EditorUserBuildSettings.development && !sentryCliOptions.UploadDevelopmentSymbols)`
-                // {
-                //     logger.LogDebug("Automated symbols upload for development builds has been disabled.");
-                //     return;
-                // }
-                //
-                // if (!sentryCliOptions.Validate(logger))
-                // {
-                //     logger.LogWarning("sentry-cli validation failed. Symbols will not be uploaded." +
-                //                        "\nYou can disable this warning by disabling the automated symbols upload under " +
-                //                        "Tools -> Sentry -> Editor");
-                //     return;
-                // }
-                //
-                // SentryCli.CreateSentryProperties(pathToProject, sentryCliOptions);
-                // SentryCli.AddExecutableToXcodeProject(pathToProject, logger);
-                //
-                // sentryXcodeProject.AddBuildPhaseSymbolUpload(logger);
             }
             catch (Exception e)
             {
-                logger.LogError("Failed to add the Sentry crash handler to the built application", e);
+                logger.LogError("Failed to add the Sentry native integration to the built application", e);
                 throw new BuildFailedException("Sentry Native BuildPostProcess failed");
             }
         }
+
+        private static void addCrashHandler(IDiagnosticLogger logger, string projectDir)
+        {
+            var crashpadPath = Path.GetFullPath(Path.Combine("Packages", SentryPackageInfo.GetName(), "Plugins",
+                "Windows", "Sentry", "crashpad_handler.exe"));
+            var targetPath = Path.Combine(projectDir, Path.GetFileName(crashpadPath));
+            logger.LogInfo($"Copying the native crash handler '{Path.GetFileName(crashpadPath)}' to the output directory");
```

Comment:
The logger takes a positional format (standard in C#) and avoids allocations if logging is disabled

```suggestion
            logger.LogInfo("Copying the native crash handler '{0}' to the output directory", Path.GetFileName(crashpadPath));
```